motor-control/firmware/stepper_motor/motor_hardware.cpp

```diff
@@ -23,6 +23,7 @@ void MotorHardware::activate_motor() {
 void MotorHardware::deactivate_motor() {
     if (pins.ebrake.has_value()) {
         gpio::set(pins.ebrake.value());
+        motor_hardware_delay(10);
```

Comment:
inadvertent inclusion?

Reply:
Not sure why this is showing up in the diff, this is whats on main right now.